imageset.py

```diff
@@ -234,6 +234,25 @@ def __getitem__(self, item):
         else:
             return self.get_corrected_data(item)
 
+    def as_experiments(self):
+        """Iterate over images in the sequence creating an experiment for
+        each with the information in the ImageSequence."""
+
+        start, end = self.get_scan().get_array_range()
+
+        from dxtbx.model.experiment_list import Experiment
```

Comment:
Import should probably be floated

Reply:
I was concerned about circular imports for this...

Reply:
I don't think that matters, and in this case you are just importing `Experiment`, which is a boost python object

Reply:
Seems to have gone away